Explorer/Assets/DCL/MapRenderer/MapLayers/PointsOfInterest/ScenesOfInterestMarkersController.cs

```diff
@@ -49,8 +49,12 @@ public async UniTask InitializeAsync(CancellationToken cancellationToken)
         {
             IReadOnlyList<string> pointsOfInterestCoordsAsync = await placesAPIService.GetPointsOfInterestCoordsAsync(cancellationToken);
             vectorCoords.Clear();
+
             foreach (var s in pointsOfInterestCoordsAsync)
-                vectorCoords.Add(IpfsHelper.DecodePointer(s));
+            {
+                try { vectorCoords.Add(IpfsHelper.DecodePointer(s)); }
```

Comment:
Just some thoughts: Looks like the issue is in how `s` is composed to throw an error in `DecodePointer`, it might be exposing an issue somewhere else? Maybe we should sanitize the values added to `pointsOfInterestsCoordsAsync` instead of try-catching a seemingly simple add?

Reply:
I think this is alright, those coordinates are coming from the backend that has the list of POIs, we should think that what it's provided is correct instead of having some additional validation on our side.
Ofc failsafe if something is off but if we add validation on our side and then something changes on the backend we risk getting "desynced" as validation rules, instead with the try we just fail if something is off and it's more noticeable.
what do you think?

Reply:
I think it's risky to blindly trust the backend, but we can accept to do it, no problem, my concern had more to do with where we do the failsafe (or sanitizing like I mentioned). Maybe doing it inside `GetPointsOfInterestCoordsAsync` might be a better place, in case some other system is using that method and might run into the same issue?
Another option would be to document that method stating that results need to be used inside a try-catch block because integrity is not guaranteed? lol